src/components/SkillEditor/SkillEditor.js

```diff
@@ -148,82 +189,227 @@ export default class Container extends React.Component {
+        var form = new FormData();
+        console.log(this.state.oldExpertValue);
+        console.log(this.state.expertValue);
+        form.append("OldModel", "general");
+        form.append("OldGroup",this.state.oldGroupValue);
+        form.append("OldLanguage", this.state.oldLanguageValue);
+        form.append("OldSkill",this.state.oldExpertValue );
+        form.append("NewModel", "general");
+        form.append("NewGroup", this.state.groupValue);
+        form.append("NewLanguage", this.state.languageValue);
+        form.append("NewSkill", this.state.expertValue);
+        form.append("changelog", this.state.commitMessage );
+        form.append("content", this.state.code);
+        form.append("imageChanged", this.state.image_name_changed);
+        form.append("old_image_name", this.state.oldImageUrl.replace("images/",""));
+        form.append("new_image_name", this.state.imageUrl.replace("images/",""));
+        form.append("image_name_changed", this.state.image_name_changed);        
+        form.append('access_token',cookies.get('loggedIn'));
 
-        console.log(url)
+        if(this.state.image_name_changed){
+            file = this.state.file;
+            form.append("image", file);
+        }
 
-            $.ajax({
-                url:url,
-                jsonpCallback: 'pc',
-                dataType: 'jsonp',
-                jsonp: 'callback',
-                crossDomain: true,
-                success: function (data) {
-                    console.log(data);
-                    if(data.accepted===true){
-                        notification.open({
-                            message: 'Accepted',
-                            description: 'Your Skill has been uploaded to the server',
-                            icon: <Icon type="check-circle" style={{ color: '#00C853' }} />,
-                        });
-                    }
-                    else{
-                        notification.open({
-                            message: 'Error Processing your Request',
-                            description: 'Please select a model, group, language and a skill',
-                            icon: <Icon type="close-circle" style={{ color: '#f44336' }} />,
-                        });
-                    }
-                },
-                error: function(e) {
-                    console.log(e);
+        var settings = {
+          "async": true,
+          "crossDomain": true,
+          "url": "http://api.susi.ai/cms/modifySkill.json",
+          "method": "POST",
+          "processData": false,
+          "contentType": false,
+          "mimeType": "multipart/form-data",
+          "data": form
+        }
+        console.log(settings);
+        for(var pair of form.entries()) {
+                console.log(pair[0]+ ', '+ pair[1]);
+              }
```

Comment:
Are you logging this for a reason? I think you can remove this for loop completely right? It's just acting as a testing procedure for you and not really doing anything else.